example/heatEquation/src/heatEquation.cpp

```diff
@@ -1,4 +1,5 @@
-/* Copyright 2019 Benjamin Worpitz, Matthias Werner
+/* Copyright 2020 Benjamin Worpitz, Matthias Werner, Jakob Krude,
+ *                Sergei Bastrakov
```

Comment:
Note: this change of the header was just forgotten in the original PR. This example was developed by @jkrude and me, based on the existing ones.